### PR TITLE
Correctly adjust mirror url

### PR DESCRIPTION
**Pull request by zeripath**

### CODE REVIEW

File: routers/repo/setting.go [hunk 1]
```diff
@@ -145,7 +151,39 @@ func SettingsPost(ctx *context.Context, form auth.RepoSettingForm) {
 				return
 			}
 		}
-		if err := ctx.Repo.Mirror.SaveAddress(form.MirrorAddress); err != nil {
+		// Validate the form.MirrorAddress
+
+		// Validate the form.MirrorAddress
```
Comment (kolaente) [inline routers/repo/setting.go:155]: Please remove the doubled comment.

**Review by kolaente — COMMENTED** [review]
Small nit, looks good otherwise.

File: routers/repo/setting.go (hunk 1 above)
Comment (zeripath) [reply routers/repo/setting.go:155]: Done.

**Review by kolaente — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]
Works with my local tests. 👍

**Pull request merged**